code/modules/telesci/telepad.dm

```diff
@@ -97,6 +112,20 @@
 	else
 		return ..()
 
+/obj/machinery/telepad_cargo/attack_hand(mob/living/user)
+	if (active == 1)
+		user << "You switch the receiver off."
+		icon_state = "pad-idle-o"
+		active = 0
+		if (cargopads.Find(src))
+			cargopads.Remove(src)
+	else
+		user << "You switch the receiver on."
+		icon_state = "pad-idle"
+		active = 1
+		if (!cargopads.Find(src))
+			cargopads.Add(src)

```

Comment:
Please use the list helpers when accessing lists, `Add` is rather slow and there are better solutions as seen here: https://github.com/tgstation/tgstation/blob/c30d07f55a1078b6463b56046884fbce9bac6959/code/__HELPERS/lists.dm